### PR TITLE
[Tests] Improve coverage for programming exercise services

### PR DESCRIPTION
**Pull request by ivanchimeno**
### Checklist
- [x] I tested *all* changes and *all* related features with different users (student, tutor, editor, instructor, admin) on the test server https://artemistest.ase.in.tum.de.
- [x] Server: I followed the [coding and design guidelines](https://docs.artemis.ase.in.tum.de/dev/guidelines/server/).
- [x] Server: I added multiple integration tests (Spring) related to the features (with a high test coverage)

### Description

This PR improves the code coverage of the `ProgrammingExerciseExportService` and `ProgrammingSubmissionService` classes to >= 90%.

### Test Coverage

ProgrammingExerciseExportService.java 90%
ProgrammingSubmissionService.java 90%

### CODE REVIEW

File: src/test/java/de/tum/in/www1/artemis/programmingexercise/ProgrammingExerciseIntegrationServiceTest.java [hunk 1]
```diff
@@ -336,16 +422,24 @@ public void textExportSubmissionsByParticipationIds_instructorNotInCourse_forbid
         request.postWithResponseBodyFile(path, getOptions(), HttpStatus.FORBIDDEN);
     }
 
-    public void textExportSubmissionsByStudentLogins() throws Exception {
+    public void testExportSubmissionsByStudentLogins() throws Exception {
+        File downloadedFile = exportSubmissionsByStudentLogins(HttpStatus.OK);
+        assertThat(downloadedFile).exists();
+        // TODO: unzip the files and add some checks
```
Comment (Olimbia23) [inline src/test/java/de/tum/in/www1/artemis/programmingexercise/ProgrammingExerciseIntegrationServiceTest.java:428]: What about this TODO? Can we remove it?

**Review by Olimbia23 — COMMENTED** [review]
Nice work @ivanchimeno. Could you fix codacy issues and then I can approve it :)
Left one small comment.

**Comment by imstreetspirit** [comment]
nice work 👍

File: src/test/java/de/tum/in/www1/artemis/programmingexercise/ProgrammingExerciseIntegrationServiceTest.java (hunk 1 above)
Comment (ivanchimeno) [reply src/test/java/de/tum/in/www1/artemis/programmingexercise/ProgrammingExerciseIntegrationServiceTest.java:428]: This was already here, I just changed the code up a bit

**Comment by ivanchimeno** [comment]
> Nice work @ivanchimeno. Could you fix codacy issues and then I can approve it :)
> Left one small comment.

These codacy issues are normal, it's due to the naming :)

**Review by ivohashamov — APPROVED** [review]
Tests look good!

**Review by Olimbia23 — APPROVED** [review]

**Review by imstreetspirit — APPROVED** [review]

**Review by joschahenningsen — APPROVED** [review]
Tests look good, thanks!

**Pull request merged**